ccd-definition/CaseEventToComplexTypes/DefendantBankAccounts.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "ID": "BankAccount",
```

Comment:
This looks exactly like the file contents in next one - ccd-definition/CaseEventToComplexTypes/Response/PartAdmit/DefendantBankAccounts.json

Reply:
I just moved both this and the one in CaseEventToFields